source/reference/partition-based-sync.txt

```diff
@@ -1020,7 +1020,18 @@ cannot express solely in JSON expressions.
        collection and then returns a boolean that indicates if the user can read
        data in the requested partition.
 
+.. _pbs-rule-migration:
 
+Migrate Partition-Based Sync Rules to Flexible Sync Rules
```

Comment:
now that flexible sync uses app services rules, I believe we should just refer to this as "App Services Rules" (I'm actually not sure what term is used in other parts of the docs)

Some links that may be relevant: 
- https://www.mongodb.com/docs/atlas/app-services/rules/roles/#with-device-sync
- https://www.mongodb.com/docs/atlas/app-services/sync/app-builder/device-sync-permissions-guide/#std-label-flexible-sync-permissions-guide

Reply:
Ah, yes! Thanks for pointing that out. I'll refactor throughout.

Reply:
We only use "App Services Rules" in a couple places in the docs. I think we default to "rules" being implicitly associated with App Services Rules. I'll double check with the team though.

Either way, I think it makes sense to call it App Services Rules here.